draft-ietf-quic-transport.md

```diff
@@ -1749,6 +1749,13 @@ values for 0-RTT.  This includes initial_max_data and either
 initial_max_streams_bidi and initial_max_stream_data_bidi_remote, or
 initial_max_streams_uni and initial_max_stream_data_uni.
 
+It is OPTIONAL for a server to be able to recover the previously sent values of
+the max_idle_timout, max_udp_payload_size, and disable_active_migration
+parameters. Lowering the values of these parameters while also accepting 0-RTT
+data could degrade the performance of the connection. Specifically, lowering the
```

Comment:
How does lowering the `max_idle_timeout` degrade the performance of the connection (unless you lower it to a value smaller than the time it takes the client to receive the new transport parameters)?

Reply:
Of the listed parameters, I _think_ `max_udp_payload_size` is the only one that if decreased could lead to performance degradation. I don't think the others have been analyzed as carefully, so I went for a (possibly over) broad statement.